test/gtest/ucm/test_dlopen/Makefile.am

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (C) Mellanox Technologies Ltd. 2001-2014.  ALL RIGHTS RESERVED.
+#
+# See file LICENSE for terms.
+#
+
+
+noinst_lib_LTLIBRARIES = \
+	libdlopen_test.la \
+	libdlopen_test2.la
+
+libdlopen_test_la_SOURCES   = libtest_dlopen.c
+libdlopen_test2_la_SOURCES  = libtest_dlopen2.c
```

Comment:
maybe the could be more meaningful name than dlopen_test2?
e.g dlopen_test_do_load.so, dlopen_test_do_mmap.so